### DISCUSSION TITLE
Анонимная таблица (передача значений)

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#320] [ОШБК!] [answered]
Здравствуйте. Может кто подскажет, вопрос следующий:

Есть анонимная таблица, из которой через API отправляются данные на remote. Когда данные приходят, то они подставляются в эту анонимную таблицу (по хешу). Вопрос, по какой возможной причине из админки все работает, а через анонимную нет?
У анонимного пользователя есть доступ ко всем нужным таблицам.
Спасибо

https://user-images.githubusercontent.com/32614108/214775761-14674a88-ec1e-4557-ae68-eeb351f02779.mp4

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Предположительно у нас ошибка — завтра будем разбираться.

**Reply by ttm-alexeypolunin** [reply]
В общем это не баг а фича.

Дело в том, что для безопасности доступ к временным таблицам возможет только от того пользователя, от которого она создана, что бы исключить атаку подбором хэша.

А когда исполняется ремоут у него там несовпадающий пользователь.

Решением мы видим добавить режим исполнения ремоута от динамического пользователя, передаваемого в параметре get или post.

Скорее всего сделаем это на сл неделе.

**Reply by tvideo** [reply]
ок, спс за подсказку, реализовал другим способом, но буду следить за обновлениями)

**Reply by ttm-alexeypolunin** [reply]
Теперь можно выбрать динамического пользователя, тогда `id` пользователя надо передать в `GET`-переменную `ttm_user` с `id` пользователя в системе от которого нужно выполнить этот ремоут.
